apps/test/unit/templates/VersionHistoryTest.jsx

```diff
@@ -0,0 +1,303 @@
+import $ from 'jquery';
+import React from 'react';
+import {mount} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from '../../util/configuredChai';
+import VersionHistory from '@cdo/apps/templates/VersionHistory';
+import VersionRow from '@cdo/apps/templates/VersionRow';
+import {sources as sourcesApi, files as filesApi} from '@cdo/apps/clientApi';
+import * as utils from '@cdo/apps/utils';
+import project from '@cdo/apps/code-studio/initApp/project';
+import firehoseClient from '@cdo/apps/lib/util/firehose';
+
+const FAKE_CURRENT_VERSION = 'current-version-id';
+const FAKE_PREVIOUS_VERSION = 'previous-version-id';
+const FAKE_VERSION_LIST_RESPONSE = {
+  responseText: JSON.stringify(
+    [
+      {
+        versionId: FAKE_CURRENT_VERSION,
+        lastModified: Date.now(),
+        isLatest: true,
+      },
+      {
+        versionId: FAKE_PREVIOUS_VERSION,
+        lastModified: Date.now(),
```

Comment:
in an abundance of caution, how about `Date.now() + 1` on this line, so we can be sure the two `lastModified` values will be different.